clang/test/CodeGenHLSL/builtins/StructuredBuffers-constructors.hlsl

```diff
@@ -5,17 +5,22 @@
 
 StructuredBuffer<float> Buf : register(t10);
 RWStructuredBuffer<float> Buf2 : register(u5, space1);
+RasterizerOrderedStructuredBuffer<float> Buf5 : register(u1, space2);
 
 // CHECK: %"class.hlsl::StructuredBuffer" = type { target("dx.RawBuffer", float, 0, 0), float }
 // CHECK: %"class.hlsl::RWStructuredBuffer" = type { target("dx.RawBuffer", float, 1, 0), float }
+// CHECK: %"class.hlsl::RasterizerOrderedStructuredBuffer" = type { target("dx.RawBuffer", float, 1, 1), float }
 
 // CHECK: @Buf = global %"class.hlsl::StructuredBuffer" zeroinitializer, align 4
 // CHECK: @Buf2 = global %"class.hlsl::RWStructuredBuffer" zeroinitializer, align 4
+// CHECK: @Buf5 = global %"class.hlsl::RasterizerOrderedStructuredBuffer" zeroinitializer, align 4
```

Comment:
Do we know why the jump from Buf2 to Buf5 instead of Buf3?

Reply:
Rather, is there a reason the variable name for this buffer went to Buf5? Do you intend to add other buffers for Buf3 / Buf4 later?

Reply:
Yes, that is correct. It is to minimize conflicts with the Append/Consume PR.